src/Compilers/CSharp/Portable/Parser/SyntaxParser.cs

```diff
@@ -673,26 +673,6 @@ private static ErrorCode GetExpectedTokenErrorCode(SyntaxKind expected, SyntaxKi
 
         protected void GetDiagnosticSpanForMissingToken(out int offset, out int width)
         {
-            // If the previous token has a trailing EndOfLineTrivia,
-            // the missing token diagnostic position is moved to the
-            // end of line containing the previous token and
-            // its width is set to zero.
-            // Otherwise the diagnostic offset and width is set
-            // to the corresponding values of the current token
-
-            var trivia = _prevTokenTrailingTrivia;
```

Comment:
>var trivia = _prevTokenTrailingTrivia; [](http://example.com/codeflow?start=12&length=38)

I am not sure how widely ```GetDiagnosticSpanForMissingToken``` is used in the parser, but this change might affect scenarios that we don't want to be affected. At the same time it doesn't feel appropriate to use ```GetDiagnosticSpanForMissingToken``` specifically to report ```ERR_InvalidExprTerm``` because that error is not about a missing token.